src/scipp/plotting/__init__.py

```diff
@@ -7,6 +7,60 @@
 from .plot import plot as _plot
 from ..utils import running_in_jupyter
 
+is_doc_build = False
+matplotlib_first_import = True
+plt = None
+
+# If we are running inside a notebook, then make plot interactive by default.
+if running_in_jupyter():
+    from IPython import get_ipython
```

Comment:
Can you elaborate? Now we are back to importing all this stuff by default?

Reply:
because the slow part is `matplotlib`. But I can also move this into the function that is called once if you like.

Reply:
I remember all the IPython imports also took significant time. Did you check? Try `python -X importtime -c 'import scipp'`

Reply:
Thanks for the useful python command.
My workstation is so slow that i could not see a difference in jupyter.
I moved everything in that function now.